### PR TITLE
Fix runbook_url not rendered correctly

### PR DESCRIPTION
**Pull request by DebakelOrakel**
## Checklist

- [ ] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [ ] Update the documentation.
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [ ] Link this PR to related issues or PRs.

### CODE REVIEW

File: component/alertrules.libsonnet [hunk 1]
```diff
@@ -77,57 +77,66 @@ local predictESStorage = {
...
       function(it) it != null,
       [
-        local r = alertpatching.filterPatchRules(
-          g, ignore_alerts, patch_alerts
-        );
-        if std.length(r.rules) > 0 then r
-        for g in esGroups
+        local r = alertpatching.filterPatchRules(g, ignore_alerts, patch_alerts);
+        local s = renderRunbookBaseURL(r, baseURL);
+        if std.length(s.rules) > 0 then s
```
Comment (simu) [inline component/alertrules.libsonnet:111]: Could be done as follows:

```suggestion
        if std.length(r.rules) > 0 then renderRunbookBaseURL(r, baseURL)
```

File: component/alertrules.libsonnet [hunk 2]
```diff
@@ -9,8 +9,8 @@ local elasticsearch = inv.parameters.openshift4_logging.components.elasticsearch
 local loki = inv.parameters.openshift4_logging.components.lokistack;
 
 
-local runbook(alertname) =
-  'https://hub.syn.tools/openshift4-logging/runbooks/%s.html' % alertname;
+local runbook(alertname) = 'https://hub.syn.tools/openshift4-logging/runbooks/%s.html' % alertname;
+local runbookLoki(alertname) = '' % alertname;
```
Comment (simu) [inline component/alertrules.libsonnet:13]: This isn't used as far as I can see?

```suggestion
```

**Review by simu — APPROVED** [review]
LGTM overall, some comments inline

File: component/alertrules.libsonnet (hunk 2 above)
Comment (DebakelOrakel) [reply component/alertrules.libsonnet:13]: I think its used with the prediction alerts of ES

Comment (simu) [reply component/alertrules.libsonnet:13]: yes the `runbook()` function is used, but `runbookLoki()` isn't used as far as I can see.

Comment (DebakelOrakel) [reply component/alertrules.libsonnet:13]: ah, missed 🤦

**Pull request merged**